src/test/java/com/powsybl/openloadflow/sa/WoodburyDcSecurityAnalysisWithActionsTest.java

```diff
@@ -0,0 +1,454 @@
+        assertEquals(network.getBranch("S_SO_2").getTerminal2().getP(), brAbs.getP2(), LoadFlowAssert.DELTA_POWER);
+
+        // Check the second operator strategy: relative change
+        network.getTwoWindingsTransformer("NE_NO_1").getPhaseTapChanger().setTapPosition(originalTapPosition - 1);
+        loadFlowRunner.run(network, parameters);
+        // Compare results
+        assertEquals(network.getLine("S_SO_2").getTerminal1().getP(), brRel.getP1(), LoadFlowAssert.DELTA_POWER);
+        assertEquals(network.getBranch("S_SO_2").getTerminal2().getP(), brRel.getP2(), LoadFlowAssert.DELTA_POWER);
+    }
+
+    @Test
+    void testFastSaDcPhaseTapChangerTapPositionChangeAdmittanceOnlyAlphaNull() {
+        Network network = PhaseControlFactory.createWithOneT2wtTwoLines();
+        // no alpha modification with pst actions
+        network.getTwoWindingsTransformer("PS1")
+                .getPhaseTapChanger()
+                .getStep(0)
+                .setAlpha(0);
+
+        List<StateMonitor> monitors = createAllBranchesMonitors(network);
+        List<Contingency> contingencies = List.of(new Contingency("L1", new BranchContingency("L1")));
+        List<Action> actions = List.of(new PhaseTapChangerTapPositionAction("pstAbsChange", "PS1", false, 0),
+                new PhaseTapChangerTapPositionAction("pstRelChange", "PS1", true, -1));
+        List<OperatorStrategy> operatorStrategies = List.of(
+                new OperatorStrategy("strategyTapAbsChange", ContingencyContext.specificContingency("L1"), new TrueCondition(), List.of("pstAbsChange")),
+                new OperatorStrategy("strategyTapRelChange", ContingencyContext.specificContingency("L1"), new TrueCondition(), List.of("pstRelChange")));
+
+        SecurityAnalysisResult result = runSecurityAnalysis(network, contingencies, monitors, securityAnalysisParameters,
+                operatorStrategies, actions, ReportNode.NO_OP);
+
+        assertNotNull(result);
+
+        OperatorStrategyResult resultAbs = getOperatorStrategyResult(result, "strategyTapAbsChange");
+        BranchResult brAbsL2 = resultAbs.getNetworkResult().getBranchResult("L2");
+        BranchResult brAbsPS1 = resultAbs.getNetworkResult().getBranchResult("PS1");
+
+        OperatorStrategyResult resultRel = getOperatorStrategyResult(result, "strategyTapRelChange");
+        BranchResult brRelL2 = resultRel.getNetworkResult().getBranchResult("L2");
+        BranchResult brRelPS1 = resultRel.getNetworkResult().getBranchResult("PS1");
+
+        // Apply contingency by hand
+        network.getLine("L1").getTerminal1().disconnect();
+        network.getLine("L1").getTerminal2().disconnect();
+        // Apply remedial action
+        network.getTwoWindingsTransformer("PS1").getPhaseTapChanger().setTapPosition(0);
+        OpenLoadFlowParameters.create(parameters).setLowImpedanceBranchMode(OpenLoadFlowParameters.LowImpedanceBranchMode.REPLACE_BY_MIN_IMPEDANCE_LINE);
```

Comment:
Why do you need to set this parameter ? (the test works also if you comment the line)